packages/virtual-keyboard/src/virtual-keyboard-new/keyboard-tray.tsx

```diff
@@ -28,7 +28,22 @@ export function KeyboardTray({ onClick }: { onClick: OnClick }) {
     const [open, setOpen] = React.useState(false);
 
     return ReactDOM.createPortal(
-        <div id="virtual-keyboard-tray" className={classNames({ open })}>
+        <div
+            id="virtual-keyboard-tray"
+            className={classNames({ open })}
+            onMouseDown={() => {
+                // The mousedown event appears to precede a blur event on a mathInput,
+                // so this access event will set the accessed timestamp to be
+                // just before the time of the blur if keyboard is clicked while the mathInput is focused.
+                onClick([{ type: "accessed", command: "" }], +new Date());
+            }}
+            onClick={() => {
+                // If the keyboard is clicked but a key is not clicked, then we send
+                // this accessed event to make sure the math input is still re-focused after the click.
+                // (The click event appears to occur after the blur event on a mathInput.)
+                onClick([{ type: "accessed", command: "" }], +new Date());
```

Comment:
Would it be better to just send a `time` with every command in the JSON, rather than making this a second argument?

Reply:
I could make the `OnClick` type be this if preferred:
```
export type OnClick = ({
    commands,
    timestamp,
}: {
    commands: KeyCommand[];
    timestamp: number;
}) => void;
```

Reply:
I think we should make `KeyCommand` have a `timestamp?: number` on it

Reply:
That is what I tried at first. But then it seemed weird as the original commands in `key.ts` don't have time stamps, and then I'd need to add timestamps to those after the fact (or at least add a time stamp to the first entry of each array). But, now that I ended up using the time stamp from the mouse down, maybe this is OK as I could leave them off the regular key presses.